website/content/docs/v0.7/Local Platforms/qemu.md

```diff
@@ -13,9 +13,8 @@ In this guide we will create a Kubernetes cluster using QEMU.
   - `CONFIG_NET_SCH_INGRESS` enabled
 - at least `CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities
 - QEMU
-- `bridge`, `static` and `firewall` CNI plugins from the [standard CNI plugins](https://github.com/containernetworking/cni), and `tc-redirect-tap` CNI plugin from the [awslabs tc-redirect-tap](https://github.com/awslabs/tc-redirect-tap) installed to `/opt/cni/bin`
+- `bridge`, `static` and `firewall` CNI plugins from the [standard CNI plugins](https://github.com/containernetworking/cni), and `tc-redirect-tap` CNI plugin from the [awslabs tc-redirect-tap](https://github.com/awslabs/tc-redirect-tap) installed to `/opt/cni/bin` (installed automatically by `talosctl`)
```

Comment:
I will probably override any existing CNI only some CNI files are missing, right?
Maybe it's fine, just don't know if it may mess up CNI for someone.

Reply:
It installs to `~/.talos/cni/bin`, so it's private location for Talos. And it checks whether plugins are missing, so they could be installed manually as well.

Reply:
maybe "installed to `/opt/cni/bin`" is wrong in this description or I just don't get it right.

Reply:
I can fix these docs up. Good point @Unix4ever . Since its Friday, and Andrey is out next week, i think we can merge and I can fix in a follow up.